src/main/java/sirius/db/mixing/query/QueryField.java

```diff
@@ -55,20 +57,50 @@ public static QueryField eq(Mapping field) {
      * @return the generated query field
      */
     public static QueryField like(Mapping field) {
-        return new QueryField(field, Mode.LIKE);
+        return new QueryField(field, Mode.LIKE, true);
     }
 
+    /**
+     * Informs the compiler, that by default, equality checks are used for this field.
+     * <p>
+     * However, if there are wildcards in the filter value, an expanding constraint can be generated.
+     * <p>
+     * Caveat: This method is case-insensitive and therefore will <b>not use an index</b> and should only be used for reasonably sized datasets.
+     *
+     * @param field the field to search in
+     * @return the generated query field
+     */
+    public static QueryField likeIgnoreCase(Mapping field) {
+        return new QueryField(field, Mode.LIKE, false);
+    }
+
+
     /**
      * Informs the compiler, that by default an expanding search like a prefix query may be generated for this field.
      * <p>
      * Depending on the underlying database an index might be used to support this query. If the database cannot
-     * execute prefix queries if may fallback to an equals constraint.
+     * execute prefix queries it may fall back to an equals-constraint.
      *
      * @param field the field to search in
      * @return the generated query field
      */
     public static QueryField startsWith(Mapping field) {
-        return new QueryField(field, Mode.PREFIX);
+        return new QueryField(field, Mode.PREFIX, true);
+    }
+
+    /**
+     * Informs the compiler, that by default an expanding search like a case-insensitive prefix query may be generated for this field.
+     * <p>
+     * Depending on the underlying database an index might be used to support this query. If the database cannot
+     * execute prefix queries it may fall back to an equals-constraint.
+     * <p>
+     * Caveat: This method is case-insensitive and therefore will <b>not use an index</b> and should only be used for reasonably sized datasets.
```

Comment:
The two sentences clash with each other concerning whether an index **might be used** or whether an index will **not be used at all**.